SoftLayer/CLI/image/import.py

```diff
@@ -18,13 +18,38 @@
 @click.option('--os-code',
               default="",
               help="The referenceCode of the operating system software"
-                   " description for the imported VHD")
+                   " description for the imported VHD, ISO, or RAW image")
+@click.option('--ibm-api-key',
```

Comment:
I'm not terribly familiar with IBM COS, but are these values obvious to someone who has used IBM COS? If not could you add some sort of info on where to get the various keys?

Reply:
I will add some links to our documentation on the ibm-api-key and the wrapped-dek.